person.rb

```diff
@@ -0,0 +1,29 @@
+class Person
+  attr_accessor :name
+  attr_reader :id, :age
+
```

Comment:
- [ ] As mentioned previously the `age` attribute should be able to be modified so kindly make sure to specify it with an `attr_accessor` along the `:name` attribute at line `2`.